src/components/overlays/TheCreateNewTacticOverlay.vue

```diff
@@ -0,0 +1,149 @@
+                return-object
+              >
+                <template v-slot:item="data">
+                  <v-list-item-avatar size="29" class="custom-list-item-avatar">
+                    <img :src="data.item.icon" :alt="data.item.name">
+                  </v-list-item-avatar>
+                  <v-list-item-content class="custom-list-item-content">
+                    <v-list-item-title v-text="data.item.name"></v-list-item-title>
+                  </v-list-item-content>
+                </template>
+              </v-autocomplete>
+            </v-card-actions>
+            <v-spacer></v-spacer>
+            <v-card-subtitle>
+              <v-btn
+                color="primary"
+                :disabled="isDisabled()"
+                @click="createTactic()"
+              >
+                Create tactic
+              </v-btn>
+            </v-card-subtitle>
+          </v-col>
+          <v-divider vertical />
+          <v-col>
+            <v-card-subtitle>
+              <v-img v-if="tactic.map.icon" :src="tactic.map.icon" max-width="200px"/>
+            </v-card-subtitle>
+            <v-card-subtitle>{{ tactic.map.name }}</v-card-subtitle>
+          </v-col>
+        </v-row>
+      </v-card>
+    </v-dialog>
+</template>
+<script lang="ts">
+import Component, { mixins } from 'vue-class-component'
+import { EventBus } from '@/event-bus'
+import { RoomGetters } from '@/store/modules/room'
+import { Action, Getter } from 'vuex-class'
+import { Api } from '@/store/modules/types'
+import { StageActions } from '@/store/modules/stage'
+import { CustomStageConfig } from '@/util/PointerEventMapper'
+import { CanvasAction } from '@/store/modules/canvas'
+import { CanvasElement, CanvasElementHistory } from '@/types/Canvas'
+import TacticWatcher from '@/mixins/TacticWatcher'
+import uuid from 'uuid'
+import { AuthenticationGetters } from '@/store/modules/authentication'
+
+@Component({
+  name: 'TheCreateNewTacticOverlay',
+  data: () => ({
+    tactic: {
+      map: {
+        name: '',
+        icon: '',
+        desc: '',
+        width: 0,
+        height: 0,
+        ratio: 0,
+        id: 0
+      },
+      name: ''
+    },
+    search: ''
+  }),
+  mixins: [TacticWatcher]
+})
+export default class CreateNewTacticOverlay extends mixins(TacticWatcher) {
+  @Getter(`room/${RoomGetters.GAME_API}`) gameApi!: Api[]
+  @Action(`stage/${StageActions.SET_CONFIG}`) setConfig!: (config: CustomStageConfig) => void
+  @Action(`canvas/${CanvasAction.SET_CANVAS_ELEMENT}`) setCanvasElements!: (canvasElements: CanvasElement[]) => void
+  @Action(`canvas/${CanvasAction.SET_CANVAS_ELEMENT_HISTORY}`) setCanvasElementsHistory!: (canvasElements: CanvasElementHistory[]) => void
+
+  overlay = false
+  created () {
+    EventBus.$on('openCreateNewTacticOverlay', () => {
+      this.overlay = !this.overlay
+    })
+  }
+
+  get maps () {
+    const mapApi: Api[] = this.gameApi.filter((api: Api) => api.name === 'wows.encyclopedia.maps')
```

Comment:
This is hardcoded for now, but so is a lot of our game related code so we will have to look at that later.